op-challenger/game/fault/trace/cannon/executor.go

```diff
@@ -141,11 +141,11 @@ func (e *Executor) generateProof(ctx context.Context, dir string, begin uint64,
 	return err
 }
 
-func preimageDir(dir string) string {
+func PreimageDir(dir string) string {
 	return filepath.Join(dir, preimagesDir)
 }
 
-func runCmd(ctx context.Context, l log.Logger, binary string, args ...string) error {
+func RunCmd(ctx context.Context, l log.Logger, binary string, args ...string) error {
```

Comment:
Likewise with `RunCmd`, is this something we can refactor into maybe a trace utils package?

Reply:
I'm thinking we could update the cannon executor file to more closely mirror the simplicity in the asterisc executor by refactoring these standalone methods out.

Reply:
Yep. Removed all the cannon op-challenger module dependency from asterisc.